packages/block-editor/src/components/block-tools/index.js

```diff
@@ -71,44 +71,51 @@ export default function BlockTools( {
 			const clientIds = getSelectedBlockClientIds();
 			if ( clientIds.length ) {
 				event.preventDefault();
+				event.stopPropagation();
```

Comment:
Why are we stopping propagation. This is usually a sign that something is wrong. You're preventing components up the tree from listening to this event.

Reply:
I'm explaining this here, https://github.com/WordPress/gutenberg/pull/53942#issuecomment-1697774962 I think we have a bug in the event bubbling code that make the event handler trigger twice for some reason unless  the event propagation is stopped. I don't understand why yet.